### PR TITLE
{bio}[foss/2018a] Minimac4 1.0.0 + libStatGen 20190330

### LINKED ISSUE #7981: Minimac4 downloads HEAD of the libStatGen github
The Minimac4 easyconfig uses the custom `install.sh` script to build Minimac4. This uses `cget` to download the HEAD of the libStatGen github repository and then build this for linking in Minimac4.

I'll be putting in a PR that separates libStatGen into a separate easyconfig and then uses CMake to build Minimac4.

### PR DESCRIPTION
**Pull request by branfosj**
fixes #7981

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6b36f6f44401b7c459fdf868644771bd for a full test report.

**Comment by boegel** [comment]
@branfosj Is it worth updating this to try and get it merged?

Needs another sync with `develop`, and perhaps a bit more love to make the tests happy...

File: easybuild/easyconfigs/l/libStatGen/libStatGen-457bf9e-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This easyconfig was created by Simon Branford of the BEAR Software team at the University of Birmingham.
+easyblock = 'MakeCp'
+
+name = 'libStatGen'
+version = '457bf9e'
```
Comment (boegel) [inline easybuild/easyconfigs/l/libStatGen/libStatGen-457bf9e-foss-2018a.eb:5]: @branfosj This should be changed to a datestamp `20190330`?

File: easybuild/easyconfigs/l/libStatGen/libStatGen-457bf9e-foss-2018a.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# hash of the current master on 30th March 2019
+fullhash = '457bf9e93d3da0e284ae35b595fbf466f38e4159'
+
+homepage = "https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen"
+description = """Useful set of classes for creating statistical genetic programs."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/statgen/%(name)s/archive/']
+sources = ['%s.tar.gz' % fullhash]
```
Comment (boegel) [inline easybuild/easyconfigs/l/libStatGen/libStatGen-457bf9e-foss-2018a.eb:15]: @branfosj Use `download_file` here to download via commit ID and rename after download?
See https://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
> @branfosj Is it worth updating this to try and get it merged?
> 
> Needs another sync with `develop`, and perhaps a bit more love to make the tests happy...

I've updated it.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/3c11d25a4e30b16fa00e94e1a4db9ead for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/2f3ac24e871fc3f7063021086ad1733d for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bber0501u03a.bb2.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/593543add0924575993a6755c307b9e1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node2424.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/f493c50ca686095a2df79159505f787d for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Pull request merged**